examples/src/main/java/com/example/JpegImageParserFuzzer.java

```diff
@@ -23,6 +23,9 @@
 // Found https://issues.apache.org/jira/browse/IMAGING-275.
 public class JpegImageParserFuzzer {
   public static void fuzzerInitialize() {
+    // When running through a Java reproducer, do not check the Java opts.
```

Comment:
Time to extract the Java opts checks into a dedicated test in `tests`?

Reply:
Good point, will do.

Reply:
Much cleaner now, thx!